### PR TITLE
Add active community to Criteria

### PR DESCRIPTION
**Pull request by ericherman**
For a project to move from Incubation to Full Stewardship, it must have
more than the code quality criteria currently enumerated, it must also
have a high enough level of dimensions such as community and process.

This commit is a first sketch of what it means to have an active community.

### CODE REVIEW

**Comment by bvhme** [comment]
Awesome @ericherman :) I love this.

All very important points. I have some reservations about how to slot this in with the rest and will make some space for a proper review on Monday if that is okay with you.

:D

**Comment by ericherman** [comment]
Sure, let's do this monday.

File: criteria/active-community.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* Software MUST be in use by multiple parties
+* Communication channels MUST exist for users AND developers
+* The roadmap SHOULD be influenced by the needs of multiple parties
+* MAY hold regular events
+
+## Why this is important
+
+* Members of active communities support each other.
+* Codebases without communities are either too narrow for stewardship by the Foundation for Public code or must continue under incubation until a self-sustaining community can be built.
```
Comment (ElenaFdR) [inline criteria/active-community.md]:
```suggestion
* Codebases without communities are either too narrow for stewardship by the Foundation for Public Code or must continue under incubation until a self-sustaining community can be built.
```

File: criteria/active-community.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## What this does not do
+
+* This does not shift responsibility for documentation, support, bug-fixing, or anything else to an unpaid community.
```
Comment (ElenaFdR) [inline criteria/active-community.md]:
```suggestion
* shift responsibility for documentation, support, bug-fixing, or anything else to an unpaid community
```

File: criteria/active-community.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+* Count known users
+* Count messages per week
+* Count commits per week
+* Count distinct commit authors per week
+* Graph bug time-to-resolution over time
+* Count bugs and feature requests opened over time
+
+## Policy makers: what you need to do
+
+* Policy makers should ensure that their organization is listed in the known users.
```
Comment (ElenaFdR) [inline criteria/active-community.md]:
```suggestion
* Make sure their organization is listed in the known users
* Encourage relevant policy makers in organizations already collaborating on a codebase to get more involved
```

File: criteria/active-community.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## Management: what you need to do
+
+* Ensure awareness of the community
+* Track and publish metrics
+
+## Developers and designers: what you need to do
+* Track and publish code metrics
+
+## Further reading
+* [Top 5 open source community metrics to track] (https://opensource.com/business/15/12/top-5-open-source-community-metrics-track)
```
Comment (bvhme) [inline criteria/active-community.md]: This markdown link is broken

```suggestion
* [Top 5 open source community metrics to track](https://opensource.com/business/15/12/top-5-open-source-community-metrics-track)
```

File: criteria/active-community.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+order: 15
+---
+# Foster an active community
```
Comment (bvhme) [inline criteria/active-community.md]: Whitespace needed before (and after) a heading, also after the front-matter.

File: criteria/active-community.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+* Graph bug time-to-resolution over time
+* Count bugs and feature requests opened over time
+
+## Policy makers: what you need to do
+
+* Policy makers should ensure that their organization is listed in the known users.
+
+## Management: what you need to do
+
+* Ensure awareness of the community
```
Comment (ElenaFdR) [inline criteria/active-community.md]: Suggestions:

Make sure devs, designers and policy people have time to contribute to the community
Prioritise community maintenance work

File: criteria/active-community.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+* Track and publish metrics
+
+## Developers and designers: what you need to do
+* Track and publish code metrics
```
Comment (bvhme) [inline criteria/active-community.md]: There should be whitespace before the bulleted list.

File: criteria/active-community.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+* Count bugs and feature requests opened over time
+
+## Policy makers: what you need to do
+
+* Policy makers should ensure that their organization is listed in the known users.
+
+## Management: what you need to do
+
+* Ensure awareness of the community
+* Track and publish metrics
```
Comment (bvhme) [inline criteria/active-community.md]: I think this point needs a bit more context; what are these metrics on and what is their goal?

File: criteria/active-community.md (hunk 6 above)
Comment (bvhme) [inline criteria/active-community.md]: I don't really understand what this means.

File: criteria/active-community.md (hunk 3 above)
Comment (bvhme) [inline criteria/active-community.md]: Are there things which policy makers can create policies for, are there ways they can reach out to others or are there cultural things they can affect.

File: criteria/active-community.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+## Why this is important
+
+* Members of active communities support each other.
+* Codebases without communities are either too narrow for stewardship by the Foundation for Public code or must continue under incubation until a self-sustaining community can be built.
+
+## What this does not do
+
+* This does not shift responsibility for documentation, support, bug-fixing, or anything else to an unpaid community.
+
+## How to test
```
Comment (bvhme) [inline criteria/active-community.md]: The 'How to test' section should explain how to test for compliance on the 'Requirements', the current things below, whilst worthwhile metrics are not tests. Perhaps they should find a home under business management? Do we want the business management do this?

File: criteria/active-community.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+* Count known users
```
Comment (bvhme) [inline criteria/active-community.md]:
```suggestion
* The codebase is used in more than 1 context
```

File: criteria/active-community.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+* Count messages per week
+* Count commits per week
+* Count distinct commit authors per week
```
Comment (bvhme) [inline criteria/active-community.md]: * There are contributors from more than 1 context

**Comment by ericherman** [comment]
The majority of these items were added as points on existing criteria and are already merged. Metrics has a kanban issue ( https://github.com/publiccodenet/standard/issues/265 ), thus this can be closed.

**Pull request closed without merging**